packages/logging-winston/src/index.js

```diff
@@ -156,9 +156,20 @@ LoggingWinston.prototype.log = function(levelName, msg, metadata, callback) {
 
   var labels = {};
 
-  // Logging proto requires that the label values be strings, so we convert
-  // using util.inspect.
-  if (is.object(metadata)) {
+  // For error messages at severity 'error' and higher, Stackdriver
+  // Error Reporting will pick up error messages if the full stack trace is
+  // included in the textPayload or the message property of the jsonPayload.
+  // https://cloud.google.com/error-reporting/docs/formatting-error-messages
+  //
+  if (is.error(metadata)) {
+    if (msg.length === 0) {
+      msg = metadata.stack;
```

Comment:
This seems to have different behavior than in Bunyan, since here 'msg' is assigned while in Bunyan 'message' is assigned. The docs say that it should be 'message', so should this be changed to match that?